example/csv/vector_polygon.csv

```diff
@@ -0,0 +1,3 @@
+prefix,location,epsg
+vector_polygon_geojson,example/geojson/WaiteOvalTest1.geojson,4326
```

Comment:
Lets hook this up so that the epsg provided here is validated against the actual data in the code.

Reply:
This is the only comment I have on the PR that is not a trivial fix. If you do not know what I am talking about please get in touch with me ASAP.

Reply:
done